app/models/reservation.rb

```diff
@@ -60,7 +71,7 @@ def self.joins_order
       .joins("LEFT JOIN orders ON orders.id = order_details.order_id")
   end
 
-  scope :not_canceled, -> { where(canceled_at: nil) }
+  scope :not_canceled, -> { where(order_details: { canceled_at: nil }) } # dubious
```

Comment:
Indubitably. Might be interesting to see if we can get this scope out of reservation and into OrderDetail itself.